app/assets/javascripts/models/beta/projectBoard.js

```diff
@@ -68,44 +68,72 @@ export const toggleColumn = (projectBoard, column, callback) => {
 }
 
 // Drag And drop utils
+const calculateNewPosition = (aboveStory, bellowStory, storyState) => {
+  const aboveStoryState = aboveStory?.state;
+  const bellowStoryState = bellowStory?.state;
+  const aboveStoryPosition = aboveStory?.newPosition;
+  const bellowStoryPosition = bellowStory?.newPosition;
+
+  if (!aboveStory) return 1;
+  if (!bellowStory) return Number(aboveStoryPosition) + 1;
```

Comment:
In order to avoid repetition, do you think we could try to convert to a number when you're assigning the variable itself?
```js
const aboveStoryPosition = Number(aboveStory?.newPosition);
```